en/docs/administer/working-with-databases.md

```diff
@@ -10,7 +10,7 @@ product pack.
 Explained below are the default databases that you will find in the
 `         database        ` directory.
 
--   **Carbon database:** `          WSO2CARBON_DB.h2.db         ` is the
+-   **Carbon database:** `          WSO2CARBON_DB.mv.db         ` is the
```

Comment:
Registry and user management now goes to shared db. Carbon DB is internal use only now. It's not configurable, hence no need to give prominence.

Reply:
done